SolutionStack/sysdig/media/src/svg/suse.svg

```diff
@@ -0,0 +1 @@
+../../../../../common/images/src/svg/suse.svg
```

Comment:
Are you sure you need this one still? Iirc, that's the old icon.

Reply:
No.  We should be using the current icon.

Reply:
Do I just find the current svg and overwrite the existing file?

Reply:
`suse-green-white.svg` right next to it should already be the current version. Albeit, I am not sure what the use case for it is right now -- it's not set as a cover image and the normal logo on the title page comes from the stylesheets themselves. So, I think it may not really be necessary at all. :0

Reply:
That image and common reference was due to prior SBP2013 stylesheet usage and docinfo.xml content. Given the new stylesheet (2021), we can
- remove common image (suse.svg)
- remove inclusion from SS/bin/setup.sh
- cleanup Kubernetes/RA/docinfo.xml "logos" section
- remove all the corresponding symlinks